e2e/adult_income/data_loader.py

```diff
@@ -19,9 +19,10 @@
 if __name__ == "__main__":
     with DataCtx() as ctx:
         _, loader = make_dataloader(train_filepath)
-        for (dense, batch_sparse_ids, target) in tqdm(loader, desc="gen batch data..."):
-            batch_data = PersiaBatch()
-            batch_data.add_non_id_type_feature([dense])
-            batch_data.add_id_type_features(batch_sparse_ids)
-            batch_data.add_label(target)
-            ctx.send_data(batch_data)
+        for (non_id_type_feature, id_type_features, label) in tqdm(
+            loader, desc="gen batch data..."
+        ):
+            persia_batch = PersiaBatch(id_type_features)
+            persia_batch.add_non_id_type_feature(non_id_type_feature)
+            persia_batch.add_label(label)
```

Comment:
If we have python class `PersiaIdTypefeatures`, and implement `PersiaIdTypefeatures.from_nparray_list` for this class, then call `persia_batch.add_id_type_feature(id_type_feature)`, would it easier for users to learn what they should pass.

Reply:
The same for `PersiaNonTypefeatures ` and `PersiaLabel`

Reply:
After offline discussion, we will do it later. https://github.com/PersiaML/PERSIA/issues/183